src/tests/JIT/HardwareIntrinsics/X86/Avx512F/ConvertToVector256Int32WithTruncation.Double.cs

```diff
@@ -0,0 +1,309 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/******************************************************************************
+ * This file is auto-generated from a template file by the GenerateTests.csx  *
+ * script in tests\src\JIT\HardwareIntrinsics\X86\Shared. In order to make    *
+ * changes, please update the corresponding template and run according to the *
+ * directions listed in the file.                                             *
+ ******************************************************************************/
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.X86._Avx512F.handwritten
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void ConvertToVector256Int32WithTruncationDouble()
+        {
+            var test = new SimpleUnaryOpTest__ConvertToVector256Int32WithTruncationDouble();
+
+            if (test.IsSupported)
+            {
+                // Validates basic functionality works, using Unsafe.Read
+                test.RunBasicScenario_UnsafeRead();
+
+                if (Avx512F.IsSupported)
```

Comment:
So no special check for 256 (just 512)?

Reply:
Right, because Avx512F implies support for the former.